lib/mongoid/fields/validators/macro.rb

```diff
@@ -104,6 +109,10 @@ def validate_options(klass, name, options)
             if !OPTIONS.include?(option) && !Fields.options.include?(option)
               raise Errors::InvalidFieldOption.new(klass, name, option, OPTIONS)
             end
+
+            if option == :type && options[option] == Symbol
+              Mongoid.logger.warn FIELD_TYPE_IS_SYMBOL
```

Comment:
let's use parenthesis to be consistent with the rest of the style.

Reply:
Sorry about that; I was thinking of `Kernel#warn` when I wrote this, but this shouldn't be written the way I did.